ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+
+This naming convention makes it possible to extend anything quickly and can be changed based on your preferences. The names of all groups and hosts must be unique.
+
+![Figure: 2](images/ansible_groups.svg)
+
+### inventory.yml
+
+The below inventory file represents two Spines and four Leafs. The nodes are defined under the groups DC1_SPINES and DC1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
+
+The hostnames specified in the inventory must exist either in DNS or in the hosts file on your Ansible host to allow successful name lookup and be able to reach the switches directly. A successful ping from the Ansible host to each inventory host verifies name resolution (e.g., ping SPINE1).
+
+Alternatively, if DNS is not available, define the variable ansible_host to be an IP address for each device.
+
+``` yaml
+--8<--
+examples/campus-fabric/inventory.yml
+--8<--
+```
+
+## Management Network
+
+In this example, a dedicated management network on port Management0 (vrf: MGMT) and an in-band management network using SVI Vlan10 (vrf: default) is configured. In-band management is easily configured with two variables under the leaf node in `DC1_FABRIC` yml.  It auto-generates an SVI and default route on each leaf node.  On the Spine nodes it will build a matching SVI for Vlan 10 and also creates a Virtual-IP for the subnet defined.
+
+``` yaml
+leaf:
+  defaults:
+    inband_management_subnet: 10.0.0.0/24
+    inband_management_vlan: 10
+```
+
+Details on this feature can be found [here](https://avd.sh/en/stable/roles/eos_designs/doc/fabric-topology-v4.0.html?h=inband#inband-management-vlan).
+
+| Node   | Management0     | Vlan10    |
+| ------ | --------------- | --------- |
+| SPINE1 | 172.100.100.101 | 10.0.0.2  |
+| SPINE2 | 172.100.100.102 | 10.0.0.3  |
+| LEAF1A | 172.100.100.103 | 10.0.0.6  |
+| LEAF1B | 172.100.100.104 | 10.0.0.7  |
+| LEAF2A | 172.100.100.105 | 10.0.0.8  |
+| LEAF3A | 172.100.100.106 | 10.0.0.9  |
+| LEAF3B | 172.100.100.107 | 10.0.0.10 |
+| LEAF3C | 172.100.100.108 | 10.0.0.11 |
+| LEAF3D | 172.100.100.109 | 10.0.0.12 |
+| LEAF3E | 172.100.100.110 | 10.0.0.13 |
+
+In Campus Networks it is uncommon to have a dedicated out-of-band management network in each IDF. You can easily disable configuring Management0 interface and vrf MGMT, by adding these variables to the DC1_LEAFS.yml group_vars.
```

Comment:
```suggestion
In Campus Networks, it is uncommon to have a dedicated out-of-band management network in each IDF. Therefore, you can easily disable configuring the Management0 interface and the management VRF by adding these variables to the `DC1_LEAFS.yml` group_vars.
```

Reply:
Not sure if we should look at highlighting each reference to a variable file in this doc. For example `DC1_LEAFS.yml`.

Reply:
https://developers.google.com/style/filenames

Reply:
I think it should be done this way...